server/commands.go

```diff
@@ -126,13 +126,13 @@ func NewServerCommands(defaultCfg *sconfig.Config) []*cli.Command {
 				},
 				&cli.StringSliceFlag{
 					Name:    common.FlagPragma,
-					Usage:   fmt.Sprintf("Specify sqlite pragma statements in pragma=value format. Pragma options: %q.", sconfig.GetAllowedPragmas()),
+					Usage:   fmt.Sprintf("Specify SQLite pragma statements in pragma=value format. Pragma options: %q.", sconfig.GetAllowedPragmas()),
```

Comment:
```suggestion
					Usage:   fmt.Sprintf("Specify SQLite [pragma statements](https://www.sqlite.org/pragma.html] in pragma=value format. Pragma options: %q.", sconfig.GetAllowedPragmas()),
```

would be nice if we could do this, and have it show up in docs but not CLI output. possible? easy / worth it?

Reply:
@feedmeapples

Reply:
i believe that's already the case https://github.com/temporalio/cli/blob/36afc181974f4a77fca287b432a457657a249bcc/helpprinter/helpprinter.go#L42C10-L42C21

Reply:
thanks!